packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -346,6 +348,47 @@ - (void)updateProps:(const Props::Shared &)props oldProps:(const Props::Shared &
     self.accessibilityElement.accessibilityElementsHidden = newViewProps.accessibilityElementsHidden;
   }
 
+  // `accessibilityShowsLargeContentViewer`
+  if (oldViewProps.accessibilityShowsLargeContentViewer != newViewProps.accessibilityShowsLargeContentViewer) {
+    self.showsLargeContentViewer = newViewProps.accessibilityShowsLargeContentViewer;
+    if (@available(iOS 13.0, *)) {
+      if (newViewProps.accessibilityShowsLargeContentViewer) {
+        self.showsLargeContentViewer = YES;
+        UILargeContentViewerInteraction *interaction = [[UILargeContentViewerInteraction alloc] init];
+        [self addInteraction:interaction];
+      } else {
+        self.showsLargeContentViewer = NO;
+      }
+    }
+  }
+
+  // `accessibilityLargeContentTitle`
+  if (oldViewProps.accessibilityLargeContentTitle != newViewProps.accessibilityLargeContentTitle) {
+    if (@available(iOS 13.0, *)) {
+      self.largeContentTitle = RCTNSStringFromStringNilIfEmpty(newViewProps.accessibilityLargeContentTitle);
+    }
+  }
+
+  // `accessibilityLargeContentImage`
+  if (oldViewProps.accessibilityLargeContentImage != newViewProps.accessibilityLargeContentImage) {
+    if (@available(iOS 13.0, *)) {
+      self.largeContentImage = nil;
+
+      NSURL *imageURL = NSURLFromImageSource(newViewProps.accessibilityLargeContentImage);
+      if (!imageURL) {
+        return;
+      }
+
+      UIImage *image = RCTImageFromLocalAssetURL(imageURL);
```

Comment:
@joevilches I'll need your opinion here.
What would be the best way to get a UIImage from an `ImageSource`?
I can't see anything about that 🙏

Reply:
I'm still interested in the answer, but actually for this PR I think I'll focus on largeContentTitle.
For the `largeContentImage` there are a lot of things to consider, like:
- Do we want to pass only local images as props, or remote images as well
- Do we want to support svg images, can be useful for apps that already use svgs are tabbar icons
I'll open a separate PR for largeContentImage 🙂

Reply:
@bacarybruno For the image question, I am not sure. You could take a look at RCTImageComponentView.mm which may give you some ideas (I took a look and was not successful :/ ). Splitting the PR seems like a good path forward. I can help find someone more knowledgable than me when your next PR comes out

Reply:
Okay thank you. Please don't hesitate to tell me if there is something left to do for this PR.

Reply:
@joevilches the linting and tests failed in the previous commit. I just [pushed an update](https://github.com/facebook/react-native/pull/45903/commits/6dafbab7b8e641240c50402c5a47542af0abd651) for that.